GiniVision/Classes/Core/GiniScreenAPICoordinator.swift

```diff
@@ -33,10 +33,11 @@ internal final class GiniScreenAPICoordinator: NSObject, Coordinator {
     var imageAnalysisNoResultsViewController: ImageAnalysisNoResultsViewController?
     var reviewViewController: ReviewViewController?
     lazy var multiPageReviewViewController: MultipageReviewViewController = {
-        let imageDocuments = self.visionDocuments.flatMap { $0 as? GiniImageDocument }
+        guard let imageDocuments = self.visionDocuments as? [GiniImageDocument] else {
+            fatalError("The MultipageReviewViewController can only handle image documents.")
```

Comment:
Fatal error will crash even in production. This might be a good thing in many cases, but I think not in this one.
You want to catch these errors while developing, but in production you have a good backup plan. You just filter the non-images and continue.

So I'd return the old code that was filtering the collection but also assert if there is a non-image.